### PR TITLE
Enable CultureInfo netstandard 1.7 APIs on Linux

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by tarekgh** [comment]
CC @jkotas @weshaggard @danmosemsft

**Comment by tarekgh** [comment]
https://github.com/dotnet/corefx/issues/11609

File: src/corefx/System.Globalization.Native/locale.cpp [hunk 1]
```diff
@@ -147,6 +147,56 @@ const char* DetectDefaultLocaleName()
...
+extern "C" int32_t GlobalizationNative_GetLocales(UChar *value, int32_t valueLength)
+{
+    int32_t totalLength = 0;
+    int32_t index = 0;
+    int32_t localeCount = uloc_countAvailable();
+    
+    if (localeCount <=  0)
+        return -1; // failed
+    
+    for (int32_t i=0; i<localeCount; i++)
```
Comment (danmosemsft) [inline src/corefx/System.Globalization.Native/locale.cpp]: nit: one space around operators?

**Comment by danmosemsft** [comment]
:shipit:

**Pull request merged**